app/services/irs_attempts_api/event.rb

```diff
@@ -0,0 +1,65 @@
+module IrsAttemptsApi
```

Comment:
do we want Irs in the name? It's "the attempts API" and they're our first/only client, right?

Reply:
Yeah, trying to make sure it is explicit this is for one group and shouldn't be used for other agencies